src/openforms/registrations/contrib/objects_api/models.py

```diff
@@ -143,6 +149,46 @@ class ObjectsAPIGroupConfig(models.Model):
         help_text=_("Default RSIN of organization, which creates the INFORMATIEOBJECT"),
     )
 
+    # Replaces informatiobjecttype_submission_report
+    iot_submission_report = models.CharField(
+        _("submission report document type description"),
+        # InformatieObjecttype.omschrijving resource attribute in Catalogi API spec
+        max_length=80,
+        blank=True,
+        help_text=_(
+            "Description of the document type in the Catalogi API to be used for the "
+            "submission report PDF. The appropriate version will automatically be "
+            "selected based on the submission timestamp and validity dates of the "
+            "document type versions."
```

Comment:
Maybe we should explicitly mention that the attribute name in Catalogi API is `omschrijving`?